cubedash/_utils.py

```diff
@@ -415,6 +437,19 @@ def dataset_shape(ds: Dataset) -> Tuple[Optional[Polygon], bool]:
     return geom, True
 
 
+def to_wrs_84(geometry: BaseGeometry, crs: Union[int, str, dict, pyproj.CRS]):
+    origin = pyproj.Proj(init=crs)
+    dest = pyproj.Proj(init="epsg:4326")
+    tranform_wrs84 = functools.partial(pyproj.transform, origin, dest)
+    new_geometry = transform(tranform_wrs84, geometry)
+    # Unwrap coordinates, if necessary
+    new_geometry = test_wrap_coordinates(new_geometry)
```

Comment:
@Kirill888  implemented something for anti-meridean handling too, which is probably better than my effort.